### PR TITLE
Feat/responsive table icon prop

### PR DESCRIPTION
**Pull request by wigqueel**
pr for getting the ability to change default icons in sort titles of the PUIResposeTable

### CODE REVIEW

**Comment by vlacher12** [comment]
Could you pls define the purpose of this PR. What problem does it solve?

**Comment by wigqueel** [comment]
> Could you pls define the purpose of this PR. What problem does it solve?

added in description

**Review by vlacher12 — CHANGES_REQUESTED** [review]

File: src/components/responsive-table/columns.tsx [hunk 1]
```diff
@@ -25,7 +27,7 @@ const Columns = ({ columns, sort, handleSort }: ColumnsProps): JSX.Element => {
 
   return (
     <thead>
-      <tr>{tableColumns}</tr>
+    <tr>{tableColumns}</tr>
```
[the remark this answers is not in the document]
Comment (wigqueel) [reply src/components/responsive-table/columns.tsx]: all done

**Comment by vlacher12** [comment]
@wigqueel Create story with custom sort icons pls

**Review by vlacher12 — APPROVED** [review]

**Review by Exiest — APPROVED** [review]

**Pull request merged**